### ISSUE TITLE
[BUG] Baldness breaks SexBound

### ISSUE THREAD

**Issue by Rattle189** [ErinaSugino/Starbound-SBR-Mod#38]
**Describe the bug**
Characters wearing any kind of cosmetic item that causes baldness prevents the character from having sex due to an Lua error.

**Log File**
Below is a truncated version of the server-side log because the error fails to appear in the client's side.
[baldnessReport.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/11997895/baldnessReport.log)

**Additional context**
No additional context available as of writing this report.

**Comment by ErinaSugino** [comment]
What cosmetic item would that be?

**Comment by Rattle189** [comment]
It was caused by Betabound's cosmetic hider `sb_emptyhead` and another mod called Armor Adapt. Apparently, the Armor Adapt mod caused the item to become a missing asset in the server because it changed the path of the sprite to `/items/armors/armorAdapt/` which isn't present in the server and causes the player to appear in the SexBound node but unable to do anything and then when any player or NPC also interacts with the SexBound node, the players can simply cancel out of it while NPCs are stuck, and when the node is 'killed' the NPC disappears.

**Comment by ErinaSugino** [comment]
So what you're saying is that you installed Armor Adapt only on the client side and not on the server side, and now stuff is obviously breaking.

**Comment by ErinaSugino** [comment]
That would require knowing what actually causes the problem. Sadly, the OP has gone silent and the last thing we could deduct is "User error in installing mods on a server"

**Comment by Rattle189** [comment]
Yes, I apologize for the late reply. It was a user's error in our server. As suggested above, would it be possible for an automatic fallback in case a user has a mod that the server does not or is it impossible to implement?

**Comment by ErinaSugino** [comment]
There is a placeholder - if sexbound can't find something, it defaults to a bald generic human male. That however doesn't help when stuff breaks because you didn't properly install mods. And checking if a mod is installed on either the client or server is impossible, as you can only execute scripts in the server or client context if your mod is running in that context, i.e. installed there. One could send a verification message from the player to the server and check if it responds (mod installed, mod code reacts) or not - but that isn't failsafe as there is no global server context, and there might not always be an entity to ping around, or that entity is incompatible to boot with like anything FU.
